packages/example-phone/test/automation/spec/call.js

```diff
@@ -3,7 +3,7 @@ import testUsers from '@ciscospark/test-helper-test-users';
 import pkg from '../../../package';
 
 
-describe(`example-phone`, () => {
+describe.skip(`example-phone call`, () => {
```

Comment:
rather than putting call, incoming-call, etc in the first describe block, can you add another nest level? the package tests pretty much always have the package name as the first describe.